Xwt.Mac/Xwt.Mac/ContextBackendHandler.cs

```diff
@@ -276,7 +276,8 @@ public override void DrawTextLayout (object backend, TextLayout layout, double x
 		{
 			CGContext ctx = ((CGContextBackend)backend).Context;
 			SetupContextForDrawing (ctx);
-			MacTextLayoutBackendHandler.Draw (ctx, Toolkit.GetBackend (layout), x, y);
+			var li = ApplicationContext.Toolkit.GetSafeBackend (layout);
```

Comment:
Sure although that's a more thorough change through the codebase so how do you want to proceed with this?